### DISCUSSION TITLE
请问你的服务是搭建在哪上的

### DISCUSSION THREAD

**Discussion by brotherbear2008** [ddiu8081/chatgpt-demo#74] [Q&A] [answered]
超级快啊

**Comment by xxyzuo** [comment]
是因为api比较快

**Comment by ddiu8081** [comment, marked as answer]
Vercel US

**Reply by MC-dusk** [reply]
~~老大我部署了一个 *.vercel.app 但国内打不开，你的 chatgpt.ddiu.me 自定义域名是怎么整的，为什么国内能打开~~

按照https://github.com/ourongxing/chatgpt-vercel 的说法，

>vercel 域名已经被墙，但 vercel 本身没有被墙，所以你绑定自己的域名就可以了

**Comment by xuxizhen** [comment]
> Vercel US

部署到Vercel，但是提问显示AI is thinking...，是因为什么网络不通吗？还是我key感觉放对了啊，谢谢大佬指导

**Comment by ddiu8081** [comment]
> > Vercel US
> 
> 部署到Vercel，但是提问显示AI is thinking...，是因为什么网络不通吗？还是我key感觉放对了啊，谢谢大佬指导

@xuxizhen 检查下报错日志。如果是自部署 Vercel 的话，需要手动在后台设置环境变量

**Comment by xuxizhen** [comment]
> > > Vercel US
> > 
> > 
> > 部署到Vercel，但是提问显示AI is thinking...，是因为什么网络不通吗？还是我key感觉放对了啊，谢谢大佬指导
> 
> @xuxizhen 检查下报错日志。如果是自部署 Vercel 的话，需要手动在后台设置环境变量

稀里糊涂搞定了，谢谢大佬
https://chatgpt123.fyi/

**Comment by pipdax** [comment]
> 

Setting->environment variables设置了key，但是依然不可以

**Comment by dofine** [comment]
> > 
> 
> Setting->environment variables设置了key，但是依然不可以

我的也是。。已经设置了api，部署在 vercel 上，提示 [POST] /api/generate reason=EDGE_FUNCTION_INVOCATION_TIMEOUT, status=504, user_error=true

**Comment by jessemoe** [comment]
> > > > Vercel US
> > > 
> > > 
> > > 部署到Vercel，但是提问显示AI is thinking...，是因为什么网络不通吗？还是我key感觉放对了啊，谢谢大佬指导
> > 
> > 
> > @xuxizhen 检查下报错日志。如果是自部署 Vercel 的话，需要手动在后台设置环境变量
> 
> 稀里糊涂搞定了，谢谢大佬 https://chatgpt123.fyi/

请问代理怎么配置的？

**Comment by Revincx** [comment]
> > > > Vercel US
> > > 
> > > 
> > > 部署到Vercel，但是提问显示AI is thinking...，是因为什么网络不通吗？还是我key感觉放对了啊，谢谢大佬指导
> > 
> > 
> > @xuxizhen 检查下报错日志。如果是自部署 Vercel 的话，需要手动在后台设置环境变量
> 
> 稀里糊涂搞定了，谢谢大佬 https://chatgpt123.fyi/

请问怎么解决的，我这里还是提示 `[POST] /api/generate reason=EDGE_FUNCTION_INVOCATION_TIMEOUT, status=504, user_error=true`

**Reply by 18661238005** [reply]
怎么解决的啊 ！[POST] /api/generate reason=EDGE_FUNCTION_INVOCATION_TIMEOUT, status=504, user_error=true  一直超时

**Comment by Revincx** [comment]
![image](https://user-images.githubusercontent.com/25524750/222942295-83260810-fbb3-4c3b-965f-48c4a72826f1.png)
破案了，帐号被禁了。

**Reply by dofine** [reply]
我的账号是正常状态，依然不行。
![image](https://user-images.githubusercontent.com/440101/222943420-d230ab8f-819b-4132-99eb-c6988cd533e4.png)

**Reply by Root-FTW** [reply]
> 我的账号是正常状态，依然不行。 ![image](https://user-images.githubusercontent.com/440101/222943420-d230ab8f-819b-4132-99eb-c6988cd533e4.png)

你是否设法解决了这个问题？

@18661238005 ?

**Reply by apegeek** [reply]
大佬们，到底在vercel 需要配置什么？ 我这里也是504

**Comment by Root-FTW** [comment]
Has anyone found a solution to this problem?

An error occurred with your deployment
EDGE_FUNCTION_INVOCATION_TIMEOUT

My subdomain in vercel was working perfect, I didn't modify anything and suddenly it didn't work anymore.

***

有没有人找到解决这个问题的办法？

我的vercel子域工作得很好，我没有修改任何东西，突然间它就不工作了。

**Reply by CODPHISH** [reply]
你解决了吗

**Comment by xuxizhen** [comment]
不知道怎么就可以了，
但是好在现在新代码，diu大佬已经能帮助一键部署了，希望大家都能部署成功😊
